docs/website/content/integrations/opa-permissions-wrapper.md

```diff
@@ -0,0 +1,15 @@
+---
+title: OPA Permissions Wrapper
+labels:
+  category: authorization
+  layer: application
+software:
+- backstage
+inventors:
+- petermacdonald
+tutorials: # optional, links to tutorials for the integration
+- https://github.com/Parsifal-M/backstage-opa-plugins/blob/main/plugins/permission-backend-module-opa-wrapper/README.md
+code: # optional, links to code for the integration
```

Comment:
You can remove the comment from here and line 10, as that's fairly obvious :)

Reply:
Woops :broom: